snowflakes/generator.go

```diff
@@ -8,14 +8,14 @@ import (
 )
 
 const (
-	CounterLen  = 13
-	InstanceLen = 7
-	CounterMask = -1 ^ (-1 << CounterLen)
+	counterLen  = 13
+	instanceLen = 7
+	counterMask = -1 ^ (-1 << counterLen)
 )
 
 var (
-	ErrNoFuture    = errors.New("Start Time cannot be set in the future.")
-	ErrBadInstance = errors.New("Instance ID must be smaller than 129")
+	errNoFuture    = errors.New("Start Time cannot be set in the future.")
```

Comment:
`iris.arke.works\forum\snowflakes\generator.go:17:30: error strings should not be capitalized or end with punctuation or a newline`